use-cases/inventory.md

```diff
@@ -1,30 +1,257 @@
+In order to synchronize products with Mews POS, use the [Get products](../operations/products.md#get-products) endpoint `GET /api/v2/products`. You can call this API operation periodically to refresh the product catalogue and make sure it is in sync with your system. The product attributes in the response include name, description, [SKU](https://en.wikipedia.org/wiki/Stock_keeping_unit), barcode, and price information.
 
-> **Editable History Window**: For a full description and links to further information, see the [Mews Glossary for Open API users](https://help.mews.com/s/article/Mews-Glossary-for-Open-API-users?language=en_US).
+#### Example request:
 
-## Periodic update
+```
+GET [PlatformAddress]/api/v2/products
+```
 
-\< Insert content here \>
+Further detail can be obtained via linked entities such as `productType` (e.g. food or beverage), `modifierSet` (product qualifiers, such as pizza toppings) and `productVariant`. These can be included in the response by using the same request but specifying the relevant include query parameters – see [Essential guide > Relationships](../essential-guide/relationships.md).
 
-\< Reference endpoints or operations />
+#### Example request:
 
-## Testing your integration
+```
+GET [PlatformAddress]/api/v2/products?include=productType,modifierSet,productVariant
+```
 
-\< Insert content here \>
+### Pagination
 
-\< Reference other content as needed />
+The response is paginated using cursor pagination. Use the `next` link to request the next page of data.
 
-## Additional help for working with the demo environment
+## Fetching sales data
+
+Use the [Get invoices](../operations/invoices.md#get-invoices) endpoint `GET /api/v2/invoices` to fetch invoices, containing order item data. Invoice attributes in the response include a description field and information about amounts, including `tax`, `total`, `subtotal`, `discount` and `tipAmount`.
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/invoices
+```
+
+Filter the invoices by date using the `filter` query parameter – see [Essential guide > Filtering](../essential-guide/filtering.md).
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/invoices?filter%createdAtGt%5D=2024-07-25T16%3A29%3A35%2B00%3A00
+```
+
+Linked to the invoice entity are:
+
+* __User__ – the customer
+* __Register__ – the cash register or outlet terminal used for the transaction
+* __Items__ – an array of invoice order items
+
+These can be selectively included in the response by using the same request but specifying the relevant include query parameters, as per [Essential guide > Relationships](../essential-guide/relationships.md). Item attributes include `productName`, `quantity`, `total` and amount breakdowns for tax and discounts. Use the register `self` link to obtain the full register information, including outlet name.
+
+#### Example request:
+
+```
+GET [PlatformAddress]/api/v2/invoices?include=invoiceItem,user,register
```

Comment:
These should be `invoiceItems`. Additionally, users can included refunded invoices as well which are called `originalInvoices`.